.gitlab-ci.yml

```diff
@@ -38,13 +45,25 @@ Docker-Image:
     - master
 
 Deploy:
-  stage: deploy
-  image: dockerhub.ebi.ac.uk/kamal/deploy-tools:0.1
-  script:
-    - sed -i "s#<DOCKER_IMAGE>#${EG_DOCKER_IMAGE}#g" ensemblgenomes_deployment.yaml
-    - kubectl apply -f ensemblgenomes_deployment.yaml
+  extends: .deploy
   # Deploy to dev env so that we can have review.ensembl.org URL
   environment:
     name : wp-hx-dev 
   only:
-    - master
\ No newline at end of file
+    - master
```

Comment:
Umm, if we still want to keep review deployments, we probably want to deploy this either for any branch other than master, or just for a specific, hypothetical `dev` branch; otherwise deployments to `wp-hx-dev` probably don't make sense. I think deploying any branch other than master should be good enough.

Reply:
Done. Added except to deploy all non-master branches to ensembl-dev namespace at WP-HX